docs/3.x/tech/tutorials/exporting-lists.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Exporting Lists and Configurations
+description: Learn how to export your lists and Configurations in CleanSpeak
```

Comment:
This tutorial only covers exporting lists, should we change the name and description?